pkg/scheduler/util/utils.go

```diff
@@ -115,13 +126,41 @@ func PatchPodStatus(cs kubernetes.Interface, old *v1.Pod, newStatus *v1.PodStatu
 		return nil
 	}
 
-	_, err = cs.CoreV1().Pods(old.Namespace).Patch(context.TODO(), old.Name, types.StrategicMergePatchType, patchBytes, metav1.PatchOptions{}, "status")
-	return err
+	backoff := wait.Backoff{
+		Duration: retryBackoffInitialDuration,
+		Factor:   retryBackoffFactor,
+		Jitter:   retryBackoffJitter,
+		Steps:    retryBackoffSteps,
+	}
+
+	patchFn := func() error {
+		_, err := cs.CoreV1().Pods(old.Namespace).Patch(context.Background(), old.Name, types.StrategicMergePatchType, patchBytes, metav1.PatchOptions{}, "status")
+		return err
+	}
+
+	isRetriableFn := func(err error) bool {
+		if typedErr, ok := err.(*errors.StatusError); ok {
+			var retryAfterSeconds int32
+			if typedErr.Status().Details != nil {
+				retryAfterSeconds = typedErr.Status().Details.RetryAfterSeconds
```

Comment:
if the api-server is unreachable, who sets this value? the client?

Reply:
If the api-server is unreachable, the error type will not be `*errors.StatusError`.
So, scheduler will wait for a backoff time and retry again.